src/TesApi.Web/appsettings.json

```diff
@@ -14,13 +14,11 @@
     "SubscriptionId": "",
     "ResourceGroup": ""
   },
-  "CosmosDb": {
-    "CosmosDbKey": "",
-    "CosmosDbEndpoint": "",
-    "AccountName": ""
-  },
-  "RetryPolicyOptions": {
```

Comment:
Are we no longer using RetryPolicyOptions?

Reply:
Actually, I discovered that this is misnamed. As written, it does absolutely nothing. To be effective, it needs to be named `RetryPolicy`, not `RetryPolicyOptions`

Reply:
done